### PR TITLE
Update Lisk SDK to v6.0.2

### LINKED ISSUE #910: Update Lisk SDK to v6.0.2
### Description

Update Lisk SDK to v6.0.2

### Acceptance Criteria

- Update Lisk SDK to v6.0.2

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?

This PR resolves #910 

### How was it solved?

- [x] Updated Lisk SDK to v6.0.2
- [x] Bump Lisk Core version to v4.0.3

### How was it tested?

Built locally and ran against the mainnet

### CODE REVIEW

**Review by Nazgolze — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ $ npm install -g lisk-core
 $ lisk-core COMMAND
 running command...
 $ lisk-core (-v|--version|version)
-lisk-core/4.0.2 darwin-arm64 node-v18.19.1
+lisk-core/4.0.3 darwin-arm64 node-v18.19.1
```
Comment (nagdahimanshu) [inline README.md:63]: I think we also need to update version in docker readme at `lisk-core/docker/README.md`.
Ref: https://github.com/LiskHQ/lisk-core/blob/910-update-lisk-sdk-to-v6_0_2/docker/README.md#run

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Review by ishantiw — APPROVED** [review]
LGTM

**Pull request merged**